Client/Frontend/Settings/FirefoxSuggestSettingsViewController.swift

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Shared
+import UIKit
+
+class FirefoxSuggestSettingsViewController: SettingsTableViewController, FeatureFlaggable {
+    init(profile: Profile) {
+        super.init(style: .grouped)
+        self.profile = profile
+        self.title = "Firefox Suggest"
```

Comment:
Oh, I was wondering about this one! It looks like all the other debug settings in `Settings / Main / Debug`, and `ExperimentsSettingsViewController`, use non-localized strings directly in the application, so I copied that pattern 😊

Since this is also a hidden debug setting, would you still like to have it in `Strings.swift`?

Reply:
uh... my bad. I totally spaced and didn't realize this was for hidden settings. Then yes, no need to have it in the `Strings.swift`

Reply:
No worries, thanks for clarifying, and glad I checked! I added comments to `FirefoxSuggestSettings` and `FirefoxSuggestSettingsViewController` to make it clear that they're hidden debug settings.